runtime-modules/content/src/permissions/mod.rs

```diff
@@ -1,20 +1,19 @@
-/*
-* The following table summarizes the permissions in the content subsystem.
-* - Actor role as columns, controller account is Tx sender.
-* - operations on a given channel (=channel=) are rows, which are basically the guards to be
-*   implemented
-* - Entries are conditions to be verified / assertions
-*
-* |                       | *Lead*                   | *Curator*                | *Member*                | *Collaborator*                   |
-* |-----------------------+--------------------------+--------------------------+-------------------------+----------------------------------|
-* | *assets mgmt*         | channel.owner is curator | curator is channel.owner | member is channel.owner | collaborators in channel.collabs |
-* | *censorship mgmt*     | channel.owner is curator | curator is channel.owner | false                   | false                            |
-* | *category mgmt*       | true                     | true                     | false                   | false                            |
-* | *collab. set mgmt*    | channel.owner is curator | curator is channel.owner | member is channel.owner | false                            |
-* | *reward account mgmt* | false                    | curator is channel.owner | member is channel.owner | false                            |
-* | *create channel*      | false                    | true                     | true                    | false                            |
-* | *delete channel*      | false                    | curator is channel.owner | member is channel.owner | false                            |
-*/
+// The following table summarizes the permissions in the content subsystem.
+// - Actor role as columns, controller account is Tx sender.
+// - operations on a given channel (=channel=) are rows, which are basically the guards to be
+//   implemented
+// - Entries are conditions to be verified / assertions
+//
+// |                       | *Lead*                   | *Curator*                | *Member*                | *Collaborator*            |
+// |-----------------------+--------------------------+--------------------------+-------------------------+---------------------------|
+// | *assets mgmt*         | channel.owner is curator | curator is channel.owner | member is channel.owner | collab in channel.collabs |
+// | *censorship mgmt*     | true                     | channel.owner is member  | false                   | false                     |
+// | *category mgmt*       | true                     | true                     | false                   | false                     |
+// | *collab. set mgmt*    | channel.owner is curator | curator is channel.owner | member is channel.owner | false                     |
+// | *reward account mgmt* | channel.owner is curator | curator is channel.owner | member is channel.owner | false                     |
```

Comment:
You need to update `ensure_actor_can_manage_reward_account` to reflect this change (allow `Lead` context)